tests/OSPSuite.SimModel.Tests/src/SimulationSpecs.cpp

```diff
@@ -267,6 +267,7 @@ namespace SimulationTests
 
     public:
         [TestAttribute]
+        [Ignore("")]
```

Comment:
deactivated all tests with band linear solver. Is not used by the OSP anyway and will be probably removed soon.
Instead, I'll try to integrate new Sparse linear solver availavle in CVODES 4.